From acc783659fb53454f296d97e7ae394cb23ca2583 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Mon, 26 Nov 2018 09:45:50 +0100
Subject: [PATCH] Merge pull request #287 in SW_UGW/linux from
 bugfix/DRVLIB_SW-823-build-dependency-of-tmu-driver-to-dp-lib to master

* commit 'ba833d1be5c7a7089d2996e8c718f8170875e5dd':
  to add datapath dependency for LTQ_TMU in Kconfig
---
 drivers/net/ethernet/lantiq/tmu/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/tmu/Kconfig b/drivers/net/ethernet/lantiq/tmu/Kconfig
index 3a0c30a341d0..100b76f8875f 100644
--- a/drivers/net/ethernet/lantiq/tmu/Kconfig
+++ b/drivers/net/ethernet/lantiq/tmu/Kconfig
@@ -4,7 +4,7 @@
 
 menuconfig LTQ_TMU
 	bool "TMU QoS Support"
-	depends on SOC_GRX500 && LTQ_CBM && GRX500_CBM
+	depends on SOC_GRX500 && LTQ_CBM && GRX500_CBM && LTQ_DATAPATH
 	default y
 	---help---
 	  TMU QoS HW Support
